miio/yeelight_dual_switch.py

```diff
@@ -0,0 +1,255 @@
+import enum
+from typing import Any, Dict
+
+import click
+
+from .click_common import EnumType, command, format_output
+from .exceptions import DeviceException
+from .miot_device import MiotDevice
+
+
+class YeelightDualControlModuleException(DeviceException):
+    pass
+
+
+class Switch(enum.Enum):
+    First = 0
+    Second = 1
+
+
+_MAPPING = {
+    # http://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:switch:0000A003:yeelink-sw1:1:0000C809
+    # First Switch (siid=2)
+    "switch_1_state": {"siid": 2, "piid": 1},  # bool
+    "switch_1_default_state": {"siid": 2, "piid": 2},  # 0 - Off, 1 - On
+    "switch_1_off_delay": {"siid": 2, "piid": 3},  # -1 - Off, [1, 43200] - delay in sec
+    # Second Switch (siid=3)
+    "switch_2_state": {"siid": 3, "piid": 1},  # bool
+    "switch_2_default_state": {"siid": 3, "piid": 2},  # 0 - Off, 1 - On
+    "switch_2_off_delay": {"siid": 3, "piid": 3},  # -1 - Off, [1, 43200] - delay in sec
+    # Extensions (siid=4)
+    "interlock": {"siid": 4, "piid": 1},  # bool
+    "flex_mode": {"siid": 4, "piid": 2},  # 0 - Off, 1 - On
+    "rc_list": {"siid": 4, "piid": 3},  # string
+    "rc_list_for_del": {"siid": 4, "piid": 4},  # string
+    "toggle": {"siid": 4, "piid": 5},  # 0 - First switch, 1 - Second switch
+}
+
+
+class DualControlModuleStatus:
+    def __init__(self, data: Dict[str, Any]) -> None:
+        """
+        Response of Yeelight Dual Control Module
+        {
+            'id': 1,
+            'result': [
+                {'did': 'switch_1_state', 'siid': 2, 'piid': 1, 'code': 0, 'value': False},
+                {'did': 'switch_1_default_state', 'siid': 2, 'piid': 2, 'code': 0, 'value': True},
+                {'did': 'switch_1_off_delay', 'siid': 2, 'piid': 3, 'code': 0, 'value': 300},
+                {'did': 'switch_2_state', 'siid': 3, 'piid': 1, 'code': 0, 'value': False},
+                {'did': 'switch_2_default_state', 'siid': 3, 'piid': 2, 'code': 0, 'value': False},
+                {'did': 'switch_2_off_delay', 'siid': 3, 'piid': 3, 'code': 0, 'value': 0},
+                {'did': 'interlock', 'siid': 4, 'piid': 1, 'code': 0, 'value': False},
+                {'did': 'flex_mode', 'siid': 4, 'piid': 2, 'code': 0, 'value': True},
+                {'did': 'rc_list', 'siid': 4, 'piid': 2, 'code': 0, 'value': '[{"mac":"9db0eb4124f8","evtid":4097,"pid":339,"beaconkey":"3691bc0679eef9596bb63abf"}]'},
+            ]
+        }
+        """
+        print(data)
+        self.data = data
+
+    @property
+    def switch_1_state(self) -> bool:
+        return bool(self.data["switch_1_state"])
```

Comment:
Could you please add docstrings to the status container properties, too? Then this is good to go!